scripts/config.pl

```diff
@@ -65,10 +39,19 @@
                               not. The value of the symbol is output if one is
                               specified in the configuration file.
     full                    - Uncomments all #define's in the configuration file
-                              excluding some reserved symbols, until the
-                              'Module configuration options' section
+                              excluding some reserved symbols and the
+                              'Module configuration options' section.
+                              This is a good basis for testing the exclusion
+                              of specific additional options.
+    veryfull                - Uncomments all #define's in the configuration file
```

Comment:
`veryfull` isn't a very descriptive name. Surely we can do better?

Reply:
With `full` and `realfull` I thought we were following the telescope naming convention... Better suggestions accepted. And let's rename `realfull` while we're at it.